test/rules/rulemsiDataObjUnlink.r

```diff
@@ -2,15 +2,14 @@ myTestRule {
 #Input parameter is:
 #  Flags in form keyword=value
 #    objPath - the data object path to remove
-#    replNum - the replica number to be removed
 #    forceFlag= - flag to remove file without transferring to trash
 #    irodsAdminRmTrash - flag to allow administrator to remove trash
 #    irodsRmTrash - flag for user to remove trash
 #Output parameter is:
 #  Status
 #Output from running the example is:
 #  Replica number 1 of file /tempZone/home/rods/sub1/objunlink2 is removed
-  msiDataObjUnlink("objPath=*SourceFile++++replNum=1",*Status);
+  msiDataObjUnlink("objPath=*SourceFile",*Status);
```

Comment:
I'm not sure what this test is trying to accomplish, but we don't support `replNum` for this API anymore. I removed it, but this used to unlink the physical data. This call now moves both replicas to the trash. Is this acceptable? We don't even verify that anything happened in this test lol

Reply:
yeah, fix the comment and they're both now in the trash.

these aren't as much tests as 'example rules' for how to hold these things.   so this is fine.

Reply:
That's fair. I fixed the comments in the following commit so I think we're good. Thanks